### ISSUE TITLE
Purulent drainage from skin

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10692]
**Preferred term label:**
Purulent drainage from skin

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Herpetic whitlow
Cat-scratch disease

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001119
- label: Purulent drainage from skin
- parents: [Inflammatory abnormality of the skin (HP:0011123)](https://hpo.jax.org/app/browse/term/HP:0011123)
- pmid: 
- def: Exudation of a thick, opaque fluid from a wound or skin lesion, typically as a result of infection and taining pus, i.e., dead white blood cells, bacteria, and tissue debris.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10692
- synonyms:

**Comment by MickeySegal** [comment]
Is "taining" in the definition meant to be "draining"?

**Comment by pnrobinson** [comment]
reopening, thanks for pointing this out.

**Comment by pnrobinson** [comment]
fixed, thanks!
